test/test_functionalization.py

```diff
@@ -345,9 +331,7 @@ def f(x):
 
 def forward(self, a_1):
     ones = torch.ops.aten.ones.default([4, 2], device = device(type='cpu'), pin_memory = False)
-    view_copy = torch.ops.aten.view_copy.default(a_1, [4, 2])
-    add = torch.ops.aten.add.Tensor(a_1, ones);  ones = None
-    copy_ = torch.ops.aten.copy_.default(a_1, add);  a_1 = None
+    add = torch.ops.aten.add.Tensor(a_1, ones);  a_1 = ones = None
```

Comment:
bleh... I'll update the test so DCE doesn't run in cases where the graph we print is the result of reinplacing (we want to see the mutations in our expecttest output).